src/frontend/types/Participant.ts

```diff
@@ -1,4 +1,5 @@
 export interface Participant {
```

Comment:
in the commit message
As participants_asking_to_join and participants_in_discussion has been
addded to the video serializer, they should be defined in the frontend
video type.
=>
As participants_asking_to_join and participants_in_discussion **have** been
**added** to the video serializer, they should be defined in the frontend
video type.